sdk/documentintelligence/azure-ai-documentintelligence/CHANGELOG.md

```diff
@@ -12,16 +12,94 @@
+
+    When this feature flag is specified, the service will further extract the values of the fields specified via the 
+    `queryFields` query parameter to supplement any existing fields defined by the model as fallback.
+
+  - [Split Options](https://mixedrealitywiki.com/display/VTI/Document+Intelligence+2023-10-31-preview#DocumentIntelligence20231031preview-SplitOptions)
+
+    In the previous API versions supported by the older `azure-ai-formrecognizer` library, document splitting and 
+    classification operation (`"/documentClassifiers/{classifierId}:analyze"`) always tried to split the input file 
+    into multiple documents.
+
+    To enable a wider set of scenarios, service introduces a "split" query parameter with the new "2023-10-31-preview" 
+    service version. The following values are supported:
+
+    - `split: "auto"`
+
+      Let service determine where to split.
+
+    - `split: "none"`
+
+      The entire file is treated as a single document. No splitting is performed.
+
+    - `split: "perPage"`
+
+      Each page is treated as a separate document. Each empty page is kept as its own document.
+
+### Breaking Changes
+
+- The SDKs targeting service API version `2023-10-31-preview` have renamed the clients to
+    - `DocumentIntelligenceAdministrationClient`/`DocumentIntelligenceAdministrationAsyncClient`
+    - `DocumentIntelligenceClient`/`DocumentIntelligenceAsyncClient`
+  which is different from older `azure-ai-formrecognizer` SDKs which targeting service API version `2023-07-31` and `2022-08-31`.
+    - `DocumentAnalysisClient`/`DocumentAnalysisAsyncClient`
+    - `DocumentModelAdministrationClient`/`DocumentModelAdministrationAsyncClient`
+  
+- **prebuilt-receipt** - Currency related fields have been updated. Currency symbol ("$") and code ("USD") are returned along with the amount as shown below.
+
+  ```json
+  {
+    "content": "$123.45",
+    "confidence": 0.995,
+    "type": "currency",
+    "valueCurrency": {
+      "amount": 123.45,
+      "currencySymbol": "$",
+      "currencyCode": "USD"
+    }
+  }
+  ...
+  ```
+- API shapes have been designed from scratch to support the new Rest level client for the Document Intelligence service.
+  Please refer to the [README](https://github.com/Azure/azure-sdk-for-java/blob/main/sdk/documentintelligence/azure-ai-documentintelligence/README.md)
+  and [Samples](https://github.com/Azure/azure-sdk-for-java/blob/main/sdk/documentintelligence/azure-ai-documentintelligence/src/samples/README.md) 
+  for more understanding.
+
+### Other Changes
+
+#### Retirements/Deprecations
+
+- `"prebuilt-businessCard"` model is retired.
+- `"prebuilt-document"` model is retired, this model is essentially `"prebuilt-layout"` with `features=keyValuePairs` specified. _(This is only supported as an optional feature for "prebuilt-layout" and "prebuilt-invoice".)_
```

Comment:
`features=keyValuePairs`

Update this to how the option looks like in a JAVA code snippet